.github/workflows/playwright.yml

```diff
@@ -211,11 +217,57 @@ jobs:
 
         # List tests and execute
         echo yarn playwright test ${TFILTER:-} ${GFILTER:+-gv "$GFILTER"} -x
-        yarn playwright test ${TFILTER:-} ${GFILTER:+-gv "$GFILTER"} -x
+        yarn playwright test ${TFILTER:-} ${GFILTER:+-gv "$GFILTER"} -x | tee qase-report.log
       env:
         MODE: ${{ matrix.mode }}
-        RANCHER_URL: https://${{env.RANCHER_FQDN}}
+        RANCHER_URL: https://${{ env.RANCHER_FQDN }}
         ORIGIN: ${{ env.KUBEWARDEN }}
+        QASE_REPORT: ${{ env.QASE }}
+        QASE_APITOKEN: ${{ secrets.QASE_APITOKEN }}
+        QASE_RUN_NAME: "Run[${{github.event_name}}]: UI[${{ env.KUBEWARDEN }}] - MODE[${{ matrix.mode }}] - RANCHER[${{ matrix.rancher }}] "
+        QASE_RUN_DESCRIPTION: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+
+    # ==================================================================================================
+    # QASE Integration
+
+    # Delete previous run if we have new one
+    - name: Find previous QASE run
+      if: ${{ env.QASE && !cancelled() && github.run_attempt > 1 }}
+      uses: dawidd6/action-download-artifact@v3
+      with:
+        name: QASE_RUN_ID-${{ matrix.mode }}-${{ matrix.rancher }}
+        run_id: ${{ github.run_id }}
+        # if_no_artifact_found: ignore
```

Comment:
Remove this comment?

Reply:
it's actually useful when previous job is canceled, I uncomented